Quests/Story EP5 names/Chapter 2/quest_00300560.csv

```diff
@@ -0,0 +1,10 @@
+Title#0,"""Un amigo confiable"""
+Explanation#0,"""***"""
+StoryExplanation#0,"""[Personajes destacados]<br>Alisa, Xiera, ???<br>[Ubicación]<br>Ruinas del castillo Cuent"""
+StoryCondition#0,"""・Completar [The Crimson Castle Crusher]<br><br>La mision se desbloqueará después de completar<br>[The Crimson Castle Crusher]"""
```

Comment:
The Crimson Castle Crusher = El Tritura Castillos Carmesi
Recuerda que el nombre de la EQ esta traducido